website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,226 @@
+
+The `requirements.txt` file is placed at the top level of your dbt project directory, alongside other key files like `dbt_project.yml`:
+
+
+```shell
+
+/my_dbt_project/
+├── dbt_project.yml
+├── models/
+│   ├── my_model.sql
+├── tests/
+│   ├── my_test.sql
+└── requirements.txt
+
+```
+
+For more information on the setup of DuckDB, you can refer to [DuckDB setup](/docs/core/connect-data-platform/duckdb-setup).
+
+<Tabs>
+
+<TabItem value="local" label="Local">
+
+1. First, [clone](https://git-scm.com/docs/git-clone/en) the Jaffle Shop git repository by running the following command in your terminal:
+
+
+
+    ```bash
+    git clone https://github.com.dbt-labs/jaffle_shop_duckdb.git
+
+    ```
+
+2. Change into the docs-duckdb directory from the command line:
+
+    ```shell
+
+    cd jaffle_shop_duck_db
+
+    ```
+
+
+3. Install dbt Core and DuckDB in a virtual environment.
+
+    <Expandable alt_header="Example for Mac" >
+
+    ```shell
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
+    source venv/bin/activate
+
+    ```
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\activate.bat
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\activate.bat
+
+    ```
+
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows PowerShell" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\Activate.ps1
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\Activate.ps1
+
+    ```
+    </Expandable>
+
+
+4. Ensure your profile is setup correctly from the command line by running the following:
+
+
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+    - [dbt docs generate](/reference/commands/cmd-docs#dbt-docs-generate) &mdash; generates your project's documentation.
+    - [dbt docs serve](/reference/commands/cmd-docs#dbt-docs-serve) &mdash; starts a webserver on port 8080 to serve your documentation locally and opens the documentation site in your default browser.
+
+    For complete details, refer to the [dbt command reference](/reference/dbt-commands).
+
```

Comment:
it might be beneficial for both tabs (local and web browser) to add a screenshot or code snippet of what successful output looks like.

Reply:
```
(venv) ➜  jaffle_shop_duckdb git:(duckdb) dbt build
15:10:12  Running with dbt=1.8.1
15:10:13  Registered adapter: duckdb=1.8.1
15:10:13  Found 5 models, 3 seeds, 20 data tests, 416 macros
15:10:13  
15:10:14  Concurrency: 24 threads (target='dev')
15:10:14  
15:10:14  1 of 28 START seed file main.raw_customers ..................................... [RUN]
15:10:14  2 of 28 START seed file main.raw_orders ........................................ [RUN]
15:10:14  3 of 28 START seed file main.raw_payments ...................................... [RUN]
15:10:14  1 of 28 OK loaded seed file main.raw_customers ................................. [INSERT 100 in 0.19s]
15:10:14  2 of 28 OK loaded seed file main.raw_orders .................................... [INSERT 99 in 0.20s]
15:10:14  4 of 28 START sql view model main.stg_customers ................................ [RUN]
15:10:14  3 of 28 OK loaded seed file main.raw_payments .................................. [INSERT 113 in 0.21s]
15:10:14  5 of 28 START sql view model main.stg_orders ................................... [RUN]
15:10:14  6 of 28 START sql view model main.stg_payments ................................. [RUN]
15:10:14  4 of 28 OK created sql view model main.stg_customers ........................... [OK in 0.16s]
15:10:14  5 of 28 OK created sql view model main.stg_orders .............................. [OK in 0.15s]
15:10:14  7 of 28 START test not_null_stg_customers_customer_id .......................... [RUN]
15:10:14  8 of 28 START test unique_stg_customers_customer_id ............................ [RUN]
15:10:14  6 of 28 OK created sql view model main.stg_payments ............................ [OK in 0.16s]
15:10:14  9 of 28 START test accepted_values_stg_orders_status__placed__shipped__completed__return_pending__returned  [RUN]
15:10:14  10 of 28 START test not_null_stg_orders_order_id ............................... [RUN]
15:10:14  11 of 28 START test unique_stg_orders_order_id ................................. [RUN]
15:10:14  12 of 28 START test accepted_values_stg_payments_payment_method__credit_card__coupon__bank_transfer__gift_card  [RUN]
15:10:14  13 of 28 START test not_null_stg_payments_payment_id ........................... [RUN]
15:10:14  14 of 28 START test unique_stg_payments_payment_id ............................. [RUN]
15:10:14  7 of 28 PASS not_null_stg_customers_customer_id ................................ [PASS in 0.25s]
15:10:14  10 of 28 PASS not_null_stg_orders_order_id ..................................... [PASS in 0.22s]
15:10:14  11 of 28 PASS unique_stg_orders_order_id ....................................... [PASS in 0.22s]
15:10:14  8 of 28 PASS unique_stg_customers_customer_id .................................. [PASS in 0.26s]
15:10:14  9 of 28 PASS accepted_values_stg_orders_status__placed__shipped__completed__return_pending__returned  [PASS in 0.23s]
15:10:14  13 of 28 PASS not_null_stg_payments_payment_id ................................. [PASS in 0.22s]
15:10:14  12 of 28 PASS accepted_values_stg_payments_payment_method__credit_card__coupon__bank_transfer__gift_card  [PASS in 0.22s]
15:10:14  14 of 28 PASS unique_stg_payments_payment_id ................................... [PASS in 0.22s]
15:10:14  15 of 28 START sql table model main.customers .................................. [RUN]
15:10:14  16 of 28 START sql table model main.orders ..................................... [RUN]
15:10:14  16 of 28 OK created sql table model main.orders ................................ [OK in 0.13s]
15:10:14  17 of 28 START test accepted_values_orders_status__placed__shipped__completed__return_pending__returned  [RUN]
15:10:14  15 of 28 OK created sql table model main.customers ............................. [OK in 0.15s]
15:10:14  18 of 28 START test not_null_orders_amount ..................................... [RUN]
15:10:14  19 of 28 START test not_null_orders_bank_transfer_amount ....................... [RUN]
15:10:14  20 of 28 START test not_null_orders_coupon_amount .............................. [RUN]
15:10:14  21 of 28 START test not_null_orders_credit_card_amount ......................... [RUN]
15:10:14  22 of 28 START test not_null_orders_customer_id ................................ [RUN]
15:10:14  23 of 28 START test not_null_orders_gift_card_amount ........................... [RUN]
15:10:14  24 of 28 START test not_null_orders_order_id ................................... [RUN]
15:10:14  25 of 28 START test unique_orders_order_id ..................................... [RUN]
15:10:15  26 of 28 START test not_null_customers_customer_id ............................. [RUN]
15:10:15  27 of 28 START test relationships_orders_customer_id__customer_id__ref_customers_  [RUN]
15:10:15  28 of 28 START test unique_customers_customer_id ............................... [RUN]
15:10:15  17 of 28 PASS accepted_values_orders_status__placed__shipped__completed__return_pending__returned  [PASS in 0.32s]
15:10:15  19 of 28 PASS not_null_orders_bank_transfer_amount ............................. [PASS in 0.32s]
15:10:15  18 of 28 PASS not_null_orders_amount ........................................... [PASS in 0.32s]
15:10:15  21 of 28 PASS not_null_orders_credit_card_amount ............................... [PASS in 0.34s]
15:10:15  22 of 28 PASS not_null_orders_customer_id ...................................... [PASS in 0.34s]
15:10:15  20 of 28 PASS not_null_orders_coupon_amount .................................... [PASS in 0.34s]
15:10:15  24 of 28 PASS not_null_orders_order_id ......................................... [PASS in 0.34s]
15:10:15  23 of 28 PASS not_null_orders_gift_card_amount ................................. [PASS in 0.35s]
15:10:15  25 of 28 PASS unique_orders_order_id ........................................... [PASS in 0.35s]
15:10:15  26 of 28 PASS not_null_customers_customer_id ................................... [PASS in 0.31s]
15:10:15  28 of 28 PASS unique_customers_customer_id ..................................... [PASS in 0.31s]
15:10:15  27 of 28 PASS relationships_orders_customer_id__customer_id__ref_customers_ .... [PASS in 0.32s]
15:10:15  
15:10:15  Finished running 3 seeds, 3 view models, 20 data tests, 2 table models in 0 hours 0 minutes and 1.52 seconds (1.52s).
15:10:15  
15:10:15  Completed successfully
15:10:15  
15:10:15  Done. PASS=28 WARN=0 ERROR=0 SKIP=0 TOTAL=28
```